Installation/include/CGAL/Installation/internal/enable_third_party_libraries.h

```diff
@@ -25,13 +25,13 @@
 #  undef CGAL_USE_GMP
 #endif
 
-#if defined(__has_include)
+#if defined(__has_include) && ( ! defined _MSC_VER || _MSC_VER > 1900)
```

Comment:
What is special about MSVC 2015? It support `#pragma message` as well, according to the documentation:
https://docs.microsoft.com/en-us/cpp/preprocessor/message?view=msvc-140

Reply:
We tested that with @janetournois and the macro does not return what it should.

Reply:
on MSVC2015 only

Reply:
What is also shocking is that `defined(__has_include)` should evaluate to false, according to [this page](https://docs.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-140), because `__has_include` was only supported since VS 2017 15.3.